src/components/Settings/Settings.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactNode } from 'react';
+import { useSelector } from 'react-redux';
+
+import styled from '@emotion/styled';
+import { capitalize } from 'lodash';
+
+import { dispatch } from 'store/rematch';
+import { selectTheme } from 'store/selectors';
+import { Responsive } from 'util/helpers';
+import { V, themeNames } from 'util/theming';
+
+import GenericPanel from 'components/_General/GenericPanel';
+import Select from 'components/_General/Select';
+import Updater from './Updater';
+
+const SubsectionRoot = styled.div`
+  /* padding: 8px; */
+  width: 50%;
+  ${Responsive.below.L} {
+    width: 100%;
+  }
+  margin-bottom: 28px;
+  padding: 0 14px;
+`;
+
+const SubsectionHeader = styled.div`
+  margin-bottom: 16px;
+  font-size: 20px;
```

Comment:
Please adjust headers according to design (h2, h3 sizes). Not sure why the headers are divs?